types/bootstrap-multiselect/index.d.ts

```diff
@@ -8,12 +8,12 @@
 
 interface Templates {
     button?: string | undefined;
-    ul?: string | undefined;
+    popupContainer?: string | undefined;
```

Comment:
is this related to the jquery-highlight types? If not you should move it into a separate PR.

Reply:
yes, but I don't know how to make separate PR for it. I already sent from GitHub desktop two different PR, but for some reason they come as accumulative.